disnake/ext/commands/common_bot_base.py

```diff
@@ -591,6 +591,11 @@ def reload_extension(self, name: str, *, package: Optional[str] = None) -> None:
             sys.modules.update(modules)
             raise
 
+    def load_extensions(self, path: str) -> None:
+        """Loads all extensions in a directory."""
+        for extension in disnake.utils.search_directory(path):
+            self.load_extension(extension)
+
```

Comment:
some people might want to filter out not extensions
```suggestion
    def load_extensions(self, path: str, *, key: Callable[[str], bool] = None) -> None:
        """Loads all extensions in a directory."""
        if key is None:
            key = lambda: True
        for extension in disnake.utils.search_directory(path):
            if key(extension):
                self.load_extension(extension)
```

Reply:
In that case you should probably be using `search_directories` directly instead.